src/PackageJsonSynchronizer.php

```diff
@@ -267,8 +268,14 @@ private function updateImportMap(array $importMapEntries): void
                 $this->io->writeError(sprintf('Updating package <comment>%s</> from <info>%s</> to <info>%s</>.', $name, $version, $versionConstraint));
             }
 
+            $arguments = [];
```

Comment:
A package with version cannot be an entrypoint... so entrypoints should not be concerned here, no ?

Reply:
Yeah, i see what you mean. 

If i remember correctly, it was to avoid too many `if` levels but I can add it inside the following condition